.github/docker/ubuntu-20.04.Dockerfile

```diff
@@ -24,7 +24,6 @@ ARG BASE_DEPS="\
 
 # UMF's dependencies
 ARG UMF_DEPS="\
```

Comment:
I know we're not testing these dockers currently, but perhaps add `automake` to `MISC_DEPS` (both dockers)

Reply:
Done